projects/build-basic/src/main/resources/sbt/tox4j/lint/scalastyle-test-config.xml

```diff
@@ -0,0 +1,267 @@
+  </check>
+
+  Checks that Java @Deprecated is not used, Scala @deprecated should be used instead
+  <check level="error" class="org.scalastyle.scalariform.DeprecatedJavaChecker" enabled="true"/>
+
+  Disallow space after certain token(s)
+  <check level="error" class="org.scalastyle.scalariform.DisallowSpaceAfterTokenChecker" enabled="true"/>
+
+  Disallow space before certain token(s)
+  <check level="error" class="org.scalastyle.scalariform.DisallowSpaceBeforeTokenChecker" enabled="true"/>
+
+  If a class/trait has no members, the braces are unnecessary
+  <check level="error" class="org.scalastyle.scalariform.EmptyClassChecker" enabled="true"/>
+
+  Ensure single space after certain token(s)
+  <check level="error" class="org.scalastyle.scalariform.EnsureSingleSpaceAfterTokenChecker" enabled="true"/>
+
+  Ensure single space before certain token(s)
+  <check level="error" class="org.scalastyle.scalariform.EnsureSingleSpaceBeforeTokenChecker" enabled="true"/>
+
+  Check that if a class implements either equals or hashCode, it should implement the other
+  <check level="error" class="org.scalastyle.scalariform.EqualsHashCodeChecker" enabled="true"/>
+
+  Check that field names match a regular expression
+  XXX: Disabled due to https://github.com/scalastyle/scalastyle/issues/149.
+  <check level="error" class="org.scalastyle.scalariform.FieldNamesChecker" enabled="false">
+    <parameters>
+      <parameter name="regex"><![CDATA[[a-z][A-Za-z]*]]></parameter>
+    </parameters>
+  </check>
+
+  Checks that braces are used in for comprehensions.
+  XXX: Disabled due to https://github.com/scalastyle/scalastyle/issues/142.
+  <check level="error" class="org.scalastyle.scalariform.ForBraceChecker" enabled="false"/>
+
+  Checks that if statements have braces
+  <check level="error" class="org.scalastyle.scalariform.IfBraceChecker" enabled="true">
+    <parameters>
+      <parameter name="singleLineAllowed">true</parameter>
+      <parameter name="doubleLineAllowed">false</parameter>
+    </parameters>
+  </check>
+
+  Check that a class does not import certain classes
+  <check level="error" class="org.scalastyle.scalariform.IllegalImportsChecker" enabled="true">
+    <parameters>
+      <parameter name="illegalImports">android._,sun._,java.awt._</parameter>
+    </parameters>
+  </check>
+
+  Checks that imports are grouped together, not throughout the file
+  XXX: Disabled because it breaks path-dependent type imports.
+  <check level="error" class="org.scalastyle.scalariform.ImportGroupingChecker" enabled="false"/>
+
+  Checks that a case statement pattern match is not lower case, as this can cause confusion.
+  XXX: Disabled because it makes suggestions (add :Any or use ``) that don't apply in many cases.
+  <check level="error" class="org.scalastyle.scalariform.LowercasePatternMatchChecker" enabled="false"/>
+
+  Checks for use of magic numbers
+  <check level="error" class="org.scalastyle.scalariform.MagicNumberChecker" enabled="false">
```

Comment:
Because in unit tests, it is often more readable to have the concrete numbers, which may be arbitrary, as literals instead of as named constants.